azkaban-webserver/src/main/java/azkaban/webapp/AzkabanWebServer.java

```diff
@@ -802,6 +804,12 @@ public static void main(String[] args) throws Exception {
     Runtime.getRuntime().addShutdownHook(new Thread() {
 
       public void run() {
+        try {
+          logTopMemoryConsumers();
+        } catch (Exception e) {
```

Comment:
Should catch Throwable to catch unexpected exception so it doesn't bring down the web server